targets/TARGET_STM/TARGET_STM32F1/TARGET_BLUEPILL_F103C8/PinNames.h

```diff
@@ -37,6 +37,13 @@
 extern "C" {
 #endif
 
+typedef enum {
```

Comment:
also please fix file permission here for this file (I've missed it in the previous review)